AutomaticInterface/AutomaticInterface/InterfaceBuilder.cs

```diff
@@ -112,7 +113,24 @@ public string Build()
             foreach (var method in methodInfos)
             {
                 cb.AppendAndNormalizeMultipleLines(method.Documentation);
```

Comment:
I would probably put this foreach body into a new method. The method gets too long

Reply:
Extracted it